### PR TITLE
added age-gender-recognition-retail-0013

### PR DESCRIPTION
**Pull request by elenash**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -49,7 +50,8 @@ RUN wget  https://github.com/google/protobuf/releases/download/v3.1.0/protobuf-c
     make install && \
     rm -rf /tmp/protobuf*
 
-ARG OPENVINO_VERSION=2018.5.455
+#ARG OPENVINO_VERSION=2018.5.455
```
Comment (AlexanderDokuchaev) [inline docker/Dockerfile]: Remove line

File: README_AG.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Age & gender recognition.
+
+## Data preparation
+
+The training procedure can be done using data in HDF5 format. Please, prepare images with faces and put it into some
+<DATA_DIR> folder. Then create a special file (<DATA_FILE>) for 'train', 'val' and 'test' phase containing annotations
+with the following structure:
+    image_1_relative_path <gender> <age/100>
```
Comment (AlexanderDokuchaev) [inline README_AG.md]: Add ``` to correct render

**Comment by AlexanderDokuchaev** [comment]
@elenash  Please add link to age gender readme in https://github.com/opencv/training_toolbox_caffe/blob/develop/README.md

**Comment by elenash** [comment]
@yssemaev @snosov1 Added images. What else we need to merge?

**Review by yssemaev — COMMENTED** [review]
Code review found no issues

**Comment by snosov1** [comment]
Good to go!

**Pull request merged**